test/integration/pkg/client/resmgmt/resmgmt_queries_pvt_test.go

```diff
@@ -52,16 +54,33 @@ func TestQueryCollectionsConfig(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create new resource management client: %s", err)
 	}
-
-	resp, err := client.QueryCollectionsConfig(orgChannelID, ccID, resmgmt.WithRetry(retry.DefaultResMgmtOpts))
+	resp, err := retry.NewInvoker(retry.New(retry.TestRetryOpts)).Invoke(
```

Comment:
What does this fix?

Reply:
The log can be found in here[https://dev.azure.com/Hyperledger/Fabric-SDK-Go/_build/results?buildId=19076&view=results](url). So I add some retry too.